<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogParamView</class>
 <widget class="QDialog" name="DialogParamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>379</width>
      <height>219</height>
     </rect>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEditParam1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditParam0">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelHeader">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter parameters:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelParam0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>@INTEGER0@</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelParam1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>@TEXT0@</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelParam2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>@TEXT2@</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditParam2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelParam3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>@TEXT3@</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditParam3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelParam4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>@TEXT4@</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditParam4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogParamView</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
